PowerShell/Add-GraphPermissionsToManagedIdentity/Add-GraphPermissionsToManagedIdentity.md

```diff
@@ -34,9 +41,11 @@ The following steps are executed:
 ### EXAMPLE 1
 This example will add the Sites.ReadWrite.All and Directory.ReadWrite.All app roles to the managed identity with the name test on the tenant with id xxxx-xxxx-xxxx
 ```powershell
```

Comment:
I would split this up in 2 examples and indicate that one is using the Azure AD module and the other one the Az module